misk-admin/web/tabs/web-actions/src/rewrite/WebActionSendRequest.tsx

```diff
@@ -54,6 +56,7 @@ export default function WebActionSendRequest({ webActionMetadata }: Props) {
     axiosRequest
       .then(response => {
         setResponse(JSON.stringify(response.data, null, 2))
+        maybeDownloadFile(response)
```

Comment:
Thoughts on calling setResponse only if maybeDownloadFile returns false? And then modifying the response to 'Downloading file...' or something?

It should be fairly (hopefully) straight-forward to download the file async. Here's an example of as async axios-based method: https://github.com/squareup/ecom-phoenix-feather/blob/2f24cd31c09f7a65cc9bac3649330c12fef7cc60/resources/assets/js/pages/EditStaff.vue#L81

Which is simply called: https://github.com/squareup/ecom-phoenix-feather/blob/2f24cd31c09f7a65cc9bac3649330c12fef7cc60/resources/assets/js/pages/EditStaff.vue#L132